src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -685,7 +685,10 @@ class NotebookSection implements INotebookSection {
 	}
 
 	get relativeUri(): string {
-		return this.headerEl['id'];
+		if (this.headerEl['id']) {
```

Comment:
This can be shortened to just `return this.headerEl['id'] || this.headerEl['name'];`

Reply:
That's right! I'll update the PR 😄 Thanks!